File: Selenium/Activities/Activity12.java
package examples;

import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class Activity12 {

	public static void main(String[] args) {

			WebDriver driver = new FirefoxDriver();
			
			driver.get("https://training-support.net/webelements/drag-drop");
			System.out.println("The page title is : "+ driver.getTitle());
			Actions builder = new Actions(driver);
			
			WebElement ball = driver.findElement(By.xpath("//img[@id='ball']"));
			WebElement dropZone1 = driver.findElement(By.xpath("//div[@id='dropzone1']"));
			WebElement dropZone2 = driver.findElement(By.xpath("//div[@id='dropzone2']"));
			
			builder.click(ball).dragAndDrop(ball, dropZone1).build().perform();
			WebElement txtVerifyDropZone1 = driver.findElement(By.xpath("//div[@id='dropzone1']/span"));
			System.out.println("The confirmation text from dropzone 1 is : "+txtVerifyDropZone1.getText());
			
			builder.click(ball).dragAndDrop(ball, dropZone2).build().perform();
			WebElement txtVerifyDropZone2 = driver.findElement(By.xpath("//div[@id='dropzone2']/span"));
			System.out.println("The confirmation text from dropzone 2 is : "+txtVerifyDropZone2.getText());
			
			driver.quit();

	}

}

File: Selenium/Activities/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver import Keys, ActionChains

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/drag-drop")
    print(f"The page title is : {driver.title}")
    builder = webdriver.ActionChains(driver)
			
    ball = driver.find_element(By.XPATH,"//img[@id='ball']")
    dropZone1 = driver.find_element(By.XPATH,"//div[@id='dropzone1']")
    dropZone2 = driver.find_element(By.XPATH,"//div[@id='dropzone2']")
			
    builder.click(ball).drag_and_drop(ball, dropZone1).perform()
    txtVerifyDropZone1 = driver.find_element(By.XPATH,"//div[@id='dropzone1']/span")
    print(f"The confirmation text from dropzone 1 is : {txtVerifyDropZone1.text}")
			
    builder.click(ball).drag_and_drop(ball, dropZone2).perform()
    txtVerifyDropZone2 = driver.find_element(By.XPATH,"//div[@id='dropzone2']/span")
    print(f"The confirmation text from dropzone 2 is : {txtVerifyDropZone2.text}")
    driver.quit()